util/pagination/fields.js

```diff
@@ -0,0 +1,574 @@
+   */
+  setChannel(channel) {
+    this.channel = channel;
+
+    return this;
+  }
+
+  /**
+   * Sets the settings for the message sent by the client.
+   * @param {Message} [message=null] - The message object sent by the client, if there is any.
+   * @param {string} [content="Preparing..."] - The custom message content while preparing the embed.
+   * @returns {PaginationFieldsEmbed} - Instance of PaginationFieldsEmbed
+   */
+  setClientMessage(message = null, content = null) {
+    if (!content) content = "Preparing...";
+
+    Object.assign(this.clientMessage, { message, content });
+
+    return this;
+  }
+
+  /**
+   * Sets the maximum number of elements to be displayed per page.
+   * @param {number} [number=10] - Maximum number of elements to be displayed per page.
+   * @returns {PaginationFieldsEmbed} - Instance of PaginationFieldsEmbed
+   */
+  setElementsPerPage(number = 10) {
+    if (typeof number !== "number") throw new Error("setElementsPerPage() only accepts number type.");
+
+    this.elementsPerPage = number;
+
+    return this;
+  }
+
+  /**
+   * Sets the emojis used for navigation buttons.
+   * @param {NavigationButtons} [emojis={}] - An object containing customised emojis to use as navigation buttons.
+   * @returns {PaginationFieldsEmbed} - Instance of PaginationFieldsEmbed
+   */
+  setEmojis(emojis) {
+    Object.assign(this.emojis, emojis);
+
+    return this;
+  }
+
+  /**
+   * Sets to jump to a certain page upon calling PaginationFieldsEmbed.build().
+   * @param {number|string} [param=1] - The page number to jump to. As String: "back", "forward"
+   * @returns {PaginationFieldsEmbed} - Instance of PaginationFieldsEmbed
+   */
+  setPage(param = 1) {
+    const isString = typeof param === "string";
+
+    if (!(!isNaN(param) || isString)) throw new Error("setPage() only accepts number/string type.");
+
+    const navigator = {
+      back: this.page === 1 ? this.page : this.page - 1,
+      forward: this.page === this.pages ? this.pages : this.page + 1
+    }[param];
+
+    this.page = isString ? navigator : param;
+
+    return this;
+  }
+
+  /**
+   * Sets the time for awaiting a user action before timeout in ms.
+   * @param {number} [timeout=30000] Timeout value in ms.
+   * @returns {PaginationFieldsEmbed} - Instance of PaginationFieldsEmbed
+   */
+  setTimeout(timeout = 30000) {
+    if (typeof timeout !== "number") throw new Error("setTimeout() only accepts number type.");
+
+    this.timeout = timeout;
+
+    return this;
+  }
+
+  /**
+   * Sets whether page number indicator on embed description text is shown or not.
+   * @param {boolean} [boolean=true] - Show page indicator?
+   * @returns {PaginationFieldsEmbed} - Instance of PaginationFieldsEmbed
+   */
+  showPageIndicator(boolean = true) {
+    if (typeof boolean !== "boolean") throw new Error("showPageIndicator() only accepts boolean type.");
+
+    this.pageIndicator = boolean === true;
```

Comment:
The variable `boolean` is type boolean, enforced by a typeof check. The ` === true` is unnecesary.